tools/test-cli-fatjar-version/src/main/java/com/microsoft/playwright/testclifatjarversion/TestApp.java

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.playwright.testclifatjarversion;
+
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.Page;
+import com.microsoft.playwright.Playwright;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.util.Collections;
+
+public class TestApp {
+  public static void main(String[] args) throws IOException, InterruptedException, URISyntaxException {
+    URI uri = new URI("jar:file:/home/user/.m2/repository/com/microsoft/playwright/driver-bundle/1.28.0/driver-bundle-1.28.0.jar!/driver/linux");
```

Comment:
Playwright version should be a parameter rather than hardcoded `1.28.0` value, so that it keeps working after bumping the version. [Same approach](https://github.com/microsoft/playwright-java/blob/81d31d24f82e5910c716529f0cdd3a5c5e61764d/tools/test-cli-version/src/main/java/com/microsoft/playwright/testcliversion/TestApp.java#L9) as in test cli version should probably work.

`/home/user/` will not work on other systems. I believe the easiest way to make it version/platform independent would be to extract [these lines](https://github.com/microsoft/playwright-java/blob/81d31d24f82e5910c716529f0cdd3a5c5e61764d/driver-bundle/src/main/java/com/microsoft/playwright/impl/driver/jar/DriverJar.java#L110-L113) into a method on DriverJar and then call it here (we don't have any promises about public API of DriverJar as it is under `.impl.` package so there are no issues with exposing such method just for the test. Or alternatively you can achieve the same via reflection.

Reply:
I am not sure if I am handling the last point correctly in the latest commit, which is why I split it from the other commit handling the review comments.